src/test/kotlin/uk/gov/justice/hmpps/offenderevents/e2e/HMPPSDomainEventsIntTest.kt

```diff
@@ -47,21 +44,6 @@ class HMPPSDomainEventsIntTest : QueueListenerIntegrationTest() {
     purgeQueues()
   }
 
-  @Throws(ExecutionException::class, InterruptedException::class)
-  private fun geMessagesCurrentlyOnTestQueue(): List<String> {
-    val messageResult = prisonEventTestQueueSqsClient.receiveMessage(
-      ReceiveMessageRequest.builder().queueUrl(prisonEventTestQueueUrl).build(),
-    ).get()
-    return messageResult
-      .messages()
-      .stream()
-      .map { obj: Message -> obj.body() }
-      .map { message: String -> toSQSMessage(message) }
-      .map(SQSMessage::Message)
-      .toList()
-  }
-
-  @Throws(ExecutionException::class, InterruptedException::class)
```

Comment:
Not sure what these JVM hints were for ? Maybe due to a Java to Kotlin conversion??